frontend/src/App.js

```diff
@@ -45,6 +49,14 @@ function App() {
     </>
   ) : null;
 
+  const staffRoutes = hasRole(currentUser, "ROLE_ADMIN") || hasRole(currentUser, "ROLE_INSTRUCTOR") ? (
+    <>
+      <Route path="/courses/:courseId/staff/create" element={<StaffCreatePage />} />
+      <Route path="/courses/:courseId/staff" element={<StaffIndexPage />} />
+      <Route path="/courses/:courseId/staff/edit/:id" element={<StaffEditPage />} />

```

Comment:
```suggestion
```

There's no use case for a Staff Edit page.

Reply:
You are either a staff member for a certain course, or you aren't.   So we only have add and delete.

Reply:
Removed route